### PR TITLE
Feature/arm

### PR DESCRIPTION
**Pull request by Schlapclick**
Added arm subsystem for use

### CODE REVIEW

File: src/RobotMap.h [hunk 1]
```diff
@@ -14,13 +14,22 @@
...
+	  static const int kIDRight1Wheel = 1;
+	  static const int kIDRight2Wheel = 2;
+	  static const int kIDLeft1Wheel = 3;
+	  static const int kIDLeft2Wheel = 4;
+	//..........Intake..........
+	  static const int kIDLeftIntake = 5;
+	  static const int kIDRightIntake = 6;
+	//..........Arm..........
+	  static const int kIDLeftArm = 50;
+	  static const int kIDRightArm = 51;
```
Comment (duane701) [inline src/RobotMap.h:29]: 7 or 8 would have been just fine.

No change required.

File: src/Subsystems/Arm.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+std::shared_ptr<Arm> Arm::self;
+
+std::shared_ptr<Arm> Arm::getInstance() {
+        if (! self) {
+          self = std::shared_ptr<Arm>(new Arm);
+        }
+        return self;
+}
+
+Arm::Arm() : Subsystem("ExampleSubsystem"),
```
Comment (duane701) [inline src/Subsystems/Arm.cpp]: Should be

    +Arm::Arm() : Subsystem(kSubsystemName),

File: src/Subsystems/Arm.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#define Arm_H
+
+#include <Commands/Subsystem.h>
+#include <DoubleSolenoid.h>
+#include <ctre/phoenix.h>
+
+class Arm : public frc::Subsystem {
+private:
+	// It's desirable that everything possible under private except
+	// for methods that implement subsystem capabilities
```
Comment (duane701) [inline src/Subsystems/Arm.h:11]: We can delete all of the boiler plate comments.

No change necessary.

**Review by duane701 — COMMENTED** [review]
FTP

**Comment by Schlapclick** [comment]
Changes Have been applied

**Review by duane701 — APPROVED** [review]
LGTM

**Comment by duane701** [comment]
> Changes Have been applied

Try using "PTAL" -- Please take another look.

**Comment by duane701** [comment]
Now we need some content to the Arm subsystem. I'll bet we can come up with a couple of commands as well.

**Pull request merged**